release-please-config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "bootstrap-sha": "638cf1a666e2e05394371b2eb9223fc9d539ada0",
+  "release-type": "krm-blueprint",
+  "bump-minor-pre-major": true,
+  "include-v-in-tag": false,
+  "changelog-path": "CHANGELOG.md",
+  "packages": {
+    "solutions/landing-zone": {
+      "package-name": "solutions/landing-zone",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/guardrails": {
+      "package-name": "solutions/guardrails",
+      "tag-separator": "/",
+      "prerelease": true
+      
+    },
+    "solutions/landing-zone-v2": {
+      "package-name": "solutions/landing-zone-v2",
+      "tag-separator": "/",
+      "prerelease": true,
+      
+    },
+    "solutions/org-policies": {
+      "package-name": "solutions/org-policies",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/hierarchy": {
```

Comment:
@davelanglois-ssc @cartyc just noticed now that `solutions/hierarchy` is not an actual package, only a folder with 6 different packages inside...
most of which will be renamed in PR #276

we could remove the configs from this PR and add the 6 hierarchy packages in a later PR or wait until they're renamed with PR#276 and add them in this PR?

Reply:
I'm ok to hold off until those packages get merged in.